docs/source/reference/storage.rst

```diff
@@ -1,201 +1,131 @@
-files required by the task. Think of it as a bucket of files that can be attached
-to your task. Compared to file_mounts, storage is faster and
-can persist across runs, requiring fewer uploads from your local machine.
-Behind the scenes, storage automatically uploads all data in the source
-to a backing object store in a particular cloud (S3/GCS/R2/IBM).
+SkyPilot tasks can access data from cloud object stores such as AWS S3, Google Cloud Storage (GCS), Cloudflare R2 or IBM COS.
 
-A storage object is used by "mounting" it to a task. On mounting, the data
-specified in the source becomes available at the destination mount path.
+These objects are made available to the task at a local path on the remote VM, so
+the task can access these objects as if they were local files.
 
-A storage object can used in either :code:`MOUNT` mode or :code:`COPY` mode.
 
-* In :code:`MOUNT` mode, the backing store is directly "mounted" to the remote VM.
-  I.e., files are streamed when accessed by the task and all writes are replicated
-  to remote bucket (and any other VMs mounting the same bucket).
 
-* In :code:`COPY` mode, the files are pre-fetched and cached on the local disk.
-  Writes are not replicated on the remote store.
+Usage
+-----
 
-.. note::
-    sky.Storage does not guarantee preservation of file
-    permissions - you may need to set file permissions during task execution.
+Object stores are specified using the :code:`file_mounts` field in a SkyPilot task.
 
-Using SkyPilot Storage
-----------------------
-SkyPilot Storage can be used by specifying additional fields in the
-:code:`file_mounts`. By default, :code:`file_mounts` uses rsync to
-directly copy files from local to remote VM.
-However, you can have them backed by SkyPilot Storage, which uploads
-the files to a cloud store (e.g. S3, GCS, R2, or IBM) and have them persist there by
-specifying the :code:`name`, :code:`source` and :code:`persistent` fields. By
-enabling persistence, file_mount sync can be made significantly faster.
+.. tab-set::
 
-Here is an example of a :code:`file_mount` that uses SkyPilot Storage:
+    .. tab-item:: Use existing bucket
+        :sync: existing-bucket-tab
 
-.. code-block:: yaml
+        To access a bucket created externally (e.g., through cloud CLI or other tools),
+        specify ``source``.
 
-    file_mounts:
-      /mybucket:
-        name: my-sky-bucket # Make sure it is unique or you own this bucket name
-        source: ~/dataset # Contents of the store. Can be local or an object store path.
-        store: s3 # Could be either of [s3, gcs, r2]. Defaults to None.
-        persistent: True  # Set to False to delete the bucket after the task is done. Defaults to True.
-        mode: MOUNT  # MOUNT or COPY. Defaults to MOUNT if not specified
+        .. code-block:: yaml
 
+          # Mount an existing S3 bucket
+          file_mounts:
+            /mydata:
+                source: s3://my-bucket/ # or gs://, r2://, cos://<region>/<bucket>
```

Comment:
In all 3 tabs, can we be consistent with style of inline comments (2 spaces before `#`).